pkgs/applications/window-managers/hyprwm/hyprpicker/default.nix

```diff
@@ -24,20 +24,19 @@
 }:
 stdenv.mkDerivation (finalAttrs: {
   pname = "hyprpicker" + lib.optionalString debug "-debug";
-  version = "0.2.0";
+  version = "0.3.0";
 
   src = fetchFromGitHub {
     owner = "hyprwm";
     repo = finalAttrs.pname;
     rev = "v${finalAttrs.version}";
-    hash = "sha256-bys8S7wuY9FJRLD5WriktWED5Hi7nCKSiNbs1Rvfk4s=";
+    hash = "sha256-BYQF1zM6bJ44ag9FJ0aTSkhOTY9U7uRdp3SmRCs5fJM=";
   };
 
   cmakeBuildType = if debug then "Debug" else "Release";
 
   nativeBuildInputs = [
     cmake
-    ninja
```

Comment:
I'd recommend leaving ninja here, as upstream build instructions recommend building with Ninja, and we don't want to diverge too much.